jbi/environment.py

```diff
@@ -51,7 +43,7 @@ class Settings(BaseSettings):
     log_format: str = "json"  # set to "text" for human-readable logs
 
     # Sentry
-    sentry_dsn: Optional[SentryDsn] = None
+    sentry_dsn: Optional[AnyUrl] = None
```

Comment:
I don't know _why_ this was the fix. I'm guessing this was fallout of upgrading to Pydantic v2.